CombinatorialGames/Game/Loopy/IGame.lean

```diff
@@ -56,6 +56,8 @@ def toLGame : IGame ↪ LGame where
   toFun := toLGame'
   inj' _ _ := toLGame'_inj
 
+instance : Coe IGame LGame := ⟨toLGame⟩
```

Comment:
Thanks for adding this! I'll have to go back and clean up the code a bit after this PR.